### ISSUE TITLE
backup won't run, connects to local mysql instead of remote

### ISSUE THREAD

**Issue by MrHappy** [emoncms/backup#24]
When I try to run the backup it says it fails to connect to mysql on localhost.
For emoncms I use a mysql-database on another server.
Somehow it doesn't use the settings in settings.php og emoncms...

**Comment by TrystanLea** [comment]
Yes, it looks like we are not specifying a hostname here: https://github.com/emoncms/backup/blob/d2410e2e6b650152e03897e626d83e5737a225b7/emoncms-export.sh#L78
The backup module isn’t strictly intended for non local databases, but I will add this as an item to be improved in future

**Comment by TrystanLea** [comment]
I note now your pull requests of course, I will make a point to review this, thanks!
